gitlab-pipeline/stage/post.yml

```diff
@@ -34,11 +34,11 @@ mender-qa:failure:
   # See https://docs.coveralls.io/parallel-build-webhook
   variables:
     COVERALLS_WEBHOOK_URL: "https://coveralls.io/webhook"
-  image: curlimages/curl
+  image: curlimages/curl-base
   dependencies:
     - init:workspace
   before_script:
-    - apk --update add git xz
+    - apk --update add xz
```

Comment:
But this job requires Git, see below `git rev-parse HEAD`

Reply:
And I even searched for any usage of it! :man_facepalming: